core/domain/exp_jobs_one_off_test.py

```diff
@@ -1071,10 +1129,12 @@ def test_migration_job_audit_success(self):
             'classifier_model_id': None
         }).to_dict()
```

Comment:
If you do it like this then how would you be sure that the states_dict passed in follows the desired state schema version?

One approach you could use to have this be more independent of the current state schema version is to swap out CURRENT_STATE_SCHEMA_VERSION in tests for a fixed number. That way, your test would always try to migrate v40 to v41 (say). If you do this, be careful to use only dicts and not make use of the domain object classes, as the latter might change over time.

Reply:
Good point - wouldn't want to add to work of future development. Updated to your suggestion